oteapi_dlite/strategies/parse.py

```diff
@@ -124,7 +121,7 @@ def get(
             if cacheconfig and cacheconfig.accessKey:
                 key = cacheconfig.accessKey
             elif "key" in session:  # type: ignore
-                key = session.key  # type: ignore
+                key = session["key"]  # type: ignore
```

Comment:
key should be a property of config, and contents of session should be written to the config (and cast exception in case of conflicting values).

Reply:
It would be really good to have some conventions for this. What whould it be a good name in the configurations, `keyToDatacacheKey` ;-)